src/config/dynamic-theme-fixes.config

```diff
@@ -865,11 +865,16 @@ div.sob6jg.so1d89.mpof_ki
 a[data-analytics-click-value="SellerAllListing"]
 .mpof_ki.m389_6m > h6
 .indiana-scroll-container > ul[role="radiogroup"]
-._292b3_juwzk
 div[data-analytics-view-label="sellersOffersGroup"]
 div[data-analytics-category="allegro.container"]
 div[data-analytics-interaction-value="DELIVERY_FREEBOX"] div
 form[name="payments"]
+a[data-analytics-click-label="offerClick"]
+.role="presentation"
```

Comment:
I think you meant?
```suggestion
[role="presentation"]
```

Reply:
With [ ] does not work.

Reply:
Please merge `as is`

Reply:
I'm sorry, but could you look again? I'm quite sure that this is not an correct selector.

Reply:
It is correct. Allegro is weird sometimes. With [ ] rule didn't work. Without work fine.

Reply:
I'm sorry, but I'm quite skeptic of this selector, I've never seen such thing. Where can I find this element?